src/main/java/org/jabref/gui/preferences/customentrytypes/CustomEntryTypesTab.java

```diff
@@ -143,6 +147,26 @@ private void setupEntryTypesTable() {
 
     private void setupFieldsTable() {
         fieldNameColumn.setCellValueFactory(item -> item.getValue().nameProperty());
+        fieldNameColumn.setCellFactory(TextFieldTableCell.forTableColumn());
+        fieldNameColumn.setEditable(true);
+        fieldNameColumn.setOnEditCommit(
+                (TableColumn.CellEditEvent<FieldViewModel, String> event) -> {
+                    String newFieldValue = StringUtil.capitalizeFirst(event.getNewValue());
```

Comment:
For newcomers, this might be strange. Maybe add a comment above:

```java
// This makes the displayed name consistent to org.jabref.model.entry.field.Field#getDisplayName
```

Reply:
OK